src/components/profile/EditProfile.js

```diff
@@ -160,6 +160,8 @@ const EditProfile = ({ screenProps, theme, styles, navigation }) => {
         <UserAvatar
           profile={profile}
           onPress={handleAvatarPress}
+          size={avatarSize}
+          imageSize={avatarSize - 6}
```

Comment:
where is `6` value from ?
is it's some margin/padding from the styles ? if yes - define it as constant and reuse both in the styles and here

if you've got this value just approximately and couldn't understand where it came from - pls re-check of different resolution / device screen sizes, probably some of them will require another gap size